### DISCUSSION TITLE
How to normalize the dataset, such as dvs128gestures, in spikingjelly of patch 0.0.0.6.

### DISCUSSION THREAD

**Discussion by Ikarosy** [fangwei123456/SpikingFlow#115] [Q&A] [answered]
In spikingjelly  0.0.0.4, tutorial offers the methods to normalize dataset which are quite useful. But, after updating into 0.0.0.6, these normalization function as well as the options in class spikingjelly.datasets.dvs128_gesture.DVS128Gesture are gone.

**Comment by fangwei123456** [comment, marked as answer]
Hi, I guess most normalization used during integrating events can be implemented as `transform: Optional[Callable] = None` of the dataset?
